src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -15,105 +15,55 @@ description: Touchscreens are available on more and more devices,  from phones u
   </iframe>
 </div>
 
-Touchscreens are available on more and more devices, ranging from phones up to desktop screens. When your users choose to interact with your UI, your app should respond to their touch in intuitive and beautiful ways.
+Touchscreens are available on more and more devices, ranging from phones to
+desktop screens. When your users choose to interact with your UI, your app
+should respond to their touch in intuitive ways.
 
 <div class="clearfix"></div>
 
-## Stateful elements respond to touch 
-
-Touchscreens are available on more and more devices, ranging from phones up to desktop screens. When your users choose to interact with your UI, your app should respond to their touch in intuitive and beautiful ways.
-
-### Add Touch States
+## Respond to element states
 
 Have you ever touched or clicked an element on a web page and questioned
 whether the site actually detected it?
 
-Simply altering the color of elements as users touch parts of your UI gives a basic reassurance that your site is working. Not only does this alleviate frustation, but can also give a snappy and responsive feel to your site.
-
-#### Use Pseudo Classes to Change UI for each Touch State
-
-The fastest way to support touch is to change the UI in response to a DOM
-element’s change in state.
+Simply altering the color of an element as users touch or interact with parts
+of your UI gives a basic reassurance that your site is working. Not only does
+this alleviate frustration, it can also give a snappy and responsive feel.
 
-### TL;DR {: .hide-from-toc }
-- Make your site feel snappy and responsive: change the UI for each state <code>:hover</code>, <code>:active</code> and <code>:focus</code>.
-- Don’t override a browser’s default responses to touch and focus unless you are implementing your own UI changes.
-- Disable text selection on elements users will touch, unless there’s a good reason why users might need to copy / select the text.
-
-
-DOM elements can be in one of the following states, default, focus, hover, and active. To change
-our UI for each of these states, we need to apply styles to the following
-pseudo classes `:hover`, `:focus` and `:active` as shown below:
+DOM elements can inherit any of the following states: default, focus, hover
+and active. To change our UI for each of these states, we need to apply styles
+to the following pseudo classes `:hover`, `:focus` and `:active` as shown below:
 
 <pre class="prettyprint">
 {% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/states-example.html" region_tag="btnstates" adjust_indentation="auto" %}
 </pre>
 
-[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ui/input/touch/states-example.html){: target="_blank" .external }
```

Comment:
You need to keep the try it buttons, otherwise there's no link to the live version for people to try